patches/server/1058-Expose-LootTable-of-DecoratedPot.patch

```diff
@@ -0,0 +1,57 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: FireInstall <kettnerl@hu-berlin.de>
+Date: Fri, 29 Dec 2023 14:15:26 +0100
+Subject: [PATCH] Expose LootTable of DecoratedPot
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/block/CraftDecoratedPot.java b/src/main/java/org/bukkit/craftbukkit/block/CraftDecoratedPot.java
+index 6334774b225dbb74f1dc2d81f70913b02f6080f1..b7c685ce6e65e6e0ed3cb96562dbc6ebd7d85f6e 100644
+--- a/src/main/java/org/bukkit/craftbukkit/block/CraftDecoratedPot.java
++++ b/src/main/java/org/bukkit/craftbukkit/block/CraftDecoratedPot.java
+@@ -15,6 +15,7 @@ import org.bukkit.block.DecoratedPot;
+ import org.bukkit.craftbukkit.inventory.CraftInventoryDecoratedPot;
+ import org.bukkit.craftbukkit.util.CraftMagicNumbers;
+ import org.bukkit.inventory.DecoratedPotInventory;
++import org.bukkit.loot.LootTable; // Paper - expose loot table
+ 
+ public class CraftDecoratedPot extends CraftBlockEntityState<DecoratedPotBlockEntity> implements DecoratedPot {
+ 
+@@ -40,6 +41,38 @@ public class CraftDecoratedPot extends CraftBlockEntityState<DecoratedPotBlockEn
+         return new CraftInventoryDecoratedPot(this.getTileEntity());
+     }
+ 
++    // Paper start - expose loot table
++    @Override
++    public void setLootTable(final LootTable table) {
++        this.setLootTable(table, this.getSeed());
++    }
++
++    public void setLootTable(LootTable table, long seed) { // Paper - make public since it overrides a public method
```

Comment:
add `@Override` and remove that paper comment which was probably just copied from somewhere

Reply:
Done.